### ISSUE TITLE
The output of relative permeability is problematic 

### ISSUE THREAD

**Issue by GitPaean** [OPM/opm-output#145]
For example, adding the following line to SPE1CASE2, 
```
 RPTRST
  BASIC=2 KRO KRW KRG SWAT PRES SGAS SOIL /
```

The output of KRs is always problematic like the following 
```
 4248  'GASKR   '         300 'REAL'
 4249    0.56199553E+15   0.32365058E+15   0.25250143E+15   0.15856476E+15
 4250    0.58609271E+14   0.00000000E+00   0.00000000E+00   0.00000000E+00
 4251    0.00000000E+00   0.00000000E+00   0.32365058E+15   0.26636632E+15
 4252    0.18484831E+15   0.95997619E+14   0.94232117E+13   0.00000000E+00
 4253    0.00000000E+00   0.00000000E+00   0.00000000E+00   0.00000000E+00
 4254    0.25250141E+15   0.18484831E+15   0.10912140E+15   0.23508053E+14
 4255    0.00000000E+00   0.00000000E+00   0.00000000E+00   0.00000000E+00
 4256    0.00000000E+00   0.00000000E+00   0.15856476E+15   0.95997603E+14
```

**Comment by joakim-hove** [comment]
Hmm - not really clear what problematic means in this context?

**Comment by GitPaean** [comment]
Relative permeability should be between 0 and 1.

**Comment by GitPaean** [comment]
fixed by OPM/opm-simulators#952 . closing now.
